pkgs/top-level/python-packages.nix

```diff
@@ -99,6 +99,8 @@ in {
 
   acoustics = callPackage ../development/python-modules/acoustics { };
 
+  py3to2 = callPackage ../development/python-modules/3to2 { };
+  # Left for backwards compatibility
   "3to2" = callPackage ../development/python-modules/3to2 { };
```

Comment:
should be `"3to2" = py3to2;`

Reply:
I tried that, but that attr set doesn't have rec, so it failed. Pretty sure I shouldn't add rec to the opt level python packages attr set. Any other suggestions?

Reply:
`"3to2" = self.py3to2;`